accelerator/config.c

```diff
@@ -13,6 +13,28 @@
 #include "yaml.h"
 
 #define CONFIG_LOGGER "config"
+#define CA_PEM_LEN 1208
+#define DEFAULT_CA_PEM                                                   \
```

Comment:
How about using the default pem inside pem/?

Reply:
It's a good idea, but how can I identify the path to the default pem?
The executable file may not under `/tangle-accelerator`, which makes it hard to direct to `/pem`